### PR TITLE
Fixing bug with not setting SERVICE_BASE_URL in assisted configmap

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: scripts/deploy_assisted_service.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@ export SERVICE_NAME=assisted-service
 export NAMESPACE=${NAMESPACE:-assisted-installer}
 export SERVICE_URL=$(get_main_ip)
 export SERVICE_PORT=${SERVICE_PORT:-6000}
+export SERVICE_BASE_URL="http://${SERVICE_URL}:${SERVICE_PORT}"
```
Comment (romfreiman) [inline scripts/deploy_assisted_service.sh:11]: Why http?

Comment (tsorya) [reply scripts/deploy_assisted_service.sh:11]: cause it is still http

Comment (tsorya) [reply scripts/deploy_assisted_service.sh:11]: when we will not support it lets move to https @empovit ?

Comment (empovit) [reply scripts/deploy_assisted_service.sh:11]: Some environments can't have HTTPS. SSL termination at ingress doesn't work in minikube where port forwarding is involved, we already discussed it. I will add a flag when HTTPS support is ready.

**Pull request merged**